test/unit/dropbox.js

```diff
@@ -21,7 +21,10 @@ describe('Dropbox', () => {
     it('calls the correct request method', () => {
       const dbx = new Dropbox();
       const rpcSpy = sinon.spy(dbx, 'rpcRequest');
-      dbx.request('path', {}, USER_AUTH, 'api', RPC);
+      dbx.request('path', {}, USER_AUTH, 'api', RPC)
+        .catch((error) => {
+          // catch so there aren't any unhandled promise rejections
```

Comment:
I'm a little confused on the reasoning for catching the error but not returning anything here